assets/src/dashboard/app/views/myStories/content/storyGridView/index.js

```diff
@@ -94,19 +77,20 @@ const StoryGridView = ({
   useEffect(() => {
     if (
       !activeGridItemId &&
-      returnStoryFocusId.value &&
+      returnStoryFocusId?.value &&
       !activeGridItemIdRef.current
     ) {
       const newFocusId = returnStoryFocusId?.value;
-      itemRefs.current?.[newFocusId]?.children?.[0]?.focus();
+      setActiveGridItemId(newFocusId);
+      // grab the menu button and refocus
+      itemRefs.current?.[newFocusId]?.lastChild?.lastChild?.firstChild?.focus();
```

Comment:
could always just add a classname to the menu button and do something like:
```
itemRefs.current?.[newFocusId]?.querySelector('.story-card-menu-button')?.focus()
```

But I'm fine with this too. just may cause regressions if there's ever any update to the order of dom nodes in the card. that being said, I don't think that order has been changed in like a year and some change so 🤷

Reply:
great idea, got too deep in the weeds - thanks for suggesting